File: Python/1020.py
n = int(input())
anos = n // 365
n = n -anos * 365
meses = n // 30
n = n - meses * 30
dias = n

print("%.0d ano(s)"%anos)
print("%.0d mes(es)"%meses)
print("%.0d dia(s)"%dias)

File: JavaScript/1020.js
let input = require ('fs').readFileSync('./dev/stdin/file.txt','utf8')
let = lines = input.split('\n')
let dias = +lines.shift()
while (dias>29) {
    if (dias>=365){
        console.log(`${Math.floor(dias/365)} ano(s)`)
        dias = dias%365
    }else{
        console.log(`${Math.floor(dias/365)} ano(s)`)
    }
    if (dias >=30){
        console.log(`${Math.floor(dias/30)} mes(es)`)
        dias = dias%30
    }else{
        console.log(`${Math.floor(dias/365)} ano(s)`)
    }
}
console.log(`${dias} dia(s)`)